sway/sway.5.txt

```diff
@@ -100,11 +100,14 @@ They are expected to be used with **bindsym** or at runtime through **swaymsg**(
 **layout** toggle split::
 	Cycles between available split layouts.
 
-**move** <left|right|up|down|next|prev|first>::
-	Moves the focused container _left_, _right_, _up_, or _down_. Moving to _prev_
-	or _next_ swaps the container with its sibling in the same container. Move
-	_first_ exchanges the focused element in an auto layout with the first
-	element, i.e. promotes  the focused element to master position.
+**move** <left|right|up|down> <[px] px>::
+	Moves the focused container _left_, _right_, _up_, or _down_. If the window
+	is floating it moves it _px_ in that direction.
```

Comment:
Just proposing that two sections might make sense. If you can sufficiently clarify it without that's fine.